codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/Encoder.java

```diff
@@ -351,10 +360,6 @@ private void add(byte[] name, byte[] value) {
             remove();
         }
 
-        // Copy name and value that modifications of original do not affect the dynamic table.
```

Comment:
> if the content changes the hash will not be correct anymore.

IIUC this is was the copy operation here was to protect against. However our `Headers` implementations also doesn't protect against this either .... lets make this this fact clear on the `Encoder.encodeHeader(..)` javadocs.